content/master/concepts/pods.md

```diff
@@ -130,6 +130,49 @@ same settings for Providers and their managed resources. Applying the
 `--max-reconcile-rate` to Crossplane only controls the rate for
 core Crossplane resources. 
 {{< /hint >}}
+
+##### Enable Realtime Compositions
+
+Realtime compositions are an alpha feature. Alpha features aren't enabled by
+default.
+
+With realtime compositions enabled Crossplane will watch every composed resource
+through a Kubernetes watch. This allows Crossplane to receive events from the
+apiserver within milliseconds when a composed resource changes, e.g. because
+a provider sets the `Ready` condition to true.
```

Comment:
```suggestion
With real time compositions enabled Crossplane watches every composed resource
with a Kubernetes watch. Crossplane receives events from the
Kubernetes API server when a composed resource changes. For example, when
a provider sets the `Ready` condition to `true`.

{{<hint "important" >}}
Real time compositions are an alpha feature. Alpha features aren't enabled by
default. 
{{< /hint >}}
```

Moves the warning after the introduction to improve noticeability. 

Grammar fixes to pass vale and simplify the sentences.